src/Orckestra.Composer/Utils/Constants.cs

```diff
@@ -26,5 +27,8 @@ public static class ErrorCodes
             public const string IsOwnedByRequestedUser = "IsOwnedByRequestedUser";
             public const string IsOwnedBySomeoneElse = "IsOwnedBySomeoneElse";
         }
+
+        public const string DefaultOrderCancellationReason = "Cancelled at Customer request";
+        public const string DefaultOrderCancellationMessage = "CancelledOrder";
```

Comment:
please rename to "RequestedOrderCancellationDatePropertyBagKey" = "RequestedOrderCancellationDate"